daemon/routes/iprule/iprule.go

```diff
@@ -45,57 +46,66 @@ func (r *Router) SetupRoutingRules(
 		ipv6EnabledList = append(ipv6EnabledList, true)
 	}
 
-	defer func() { // recover if error
```

Comment:
Why was this removed?

Reply:
Ok, wrong assumption of mine, getting it back.